content/reference/v1beta1/application.md

```diff
@@ -164,3 +165,14 @@ The format of the Y axis labels with support for all Grafana [units](https://gra
 
 ### yAxisTemplate
 Y axis labels template. Use `{{ value }}`.
+
+## targetKotsVersion
+The KOTS version that is targeted by the release.
+
+Specifying this in the application spec of the release will enforce compatibility checks for new installations and block the installation if the KOTS version being used is greater than the targetted KOTS version.
+
+For channels that have the release active, the install command for existing clusters - which displayed on the channel card - will include that specific version of KOTS.
```

Comment:
The wording at the beginning of this paragraph confuses me. Is this still what you're trying to say?
```suggestion
If the latest release in a channel specifies a target KOTS version, the install command for existing clusters—which is displayed on the channel card in the vendor portal—will be modified to install that specific version of KOTS.
```